src/qt/guiutil.cpp

```diff
@@ -968,6 +968,9 @@ QString formatServicesStr(quint64 mask)
             case NODE_GRAPHENE:
                 strList.append("GRAPH");
                 break;
+            case NODE_WEAKBLOCKS:
+                strList.append("WEAKBLOCKS");
+                break;
```

Comment:
Could you maybe change the string to be something shorter...there is somewhat of a limited space in the QT peers list.  At some point we'll have to change the formatting there to better accommodate all the services there but just using something like "WB" or "WEAK", (as we do for GRAPHENE, where we truncated it a little bit.)

Reply:
Sure!